enterprise-suite/scripts/install-es.py

```diff
@@ -0,0 +1,231 @@
+                .format(tempdir, args.version or '', chart_ref))
+            chartfile_glob = tempdir + '/' + args.chart + '*.tgz'
+            chartfile = glob.glob(chartfile_glob)
+            if len(chartfile) < 1:
+                sys.exit('cannot access fetched chartfile at {}, ES_CHART={}'
+                    .format(chartfile_glob, args.chart))
+            execute('helm template --name {} --namespace {} {} {} {}'
+                .format(args.helm_name, args.namespace, helm_args,
+                credentials_arg, chartfile[0]))
+        finally:
+            shutil.rmtree(tempdir)
+
+    else:
+        # Determine if we should upgrade or install
+        should_upgrade = False
+        stdout, returncode = run('helm status ' + args.helm_name, DEFAULT_TIMEOUT)
+        if returncode == 0:
+            if args.force_install:
+                execute('helm delete --purge ' + args.helm_name)
+                print(('warning: helm delete does not wait for resources to be removed'
+                       '- if the script fails on install, please re-run it.'))
+            else:
+                should_upgrade = True
+    
+        if should_upgrade:
+            execute('helm upgrade {} {} {} {}'
+                .format(args.helm_name, chart_ref, creds_values,
+                        helm_args))
+        else:
+            execute('helm install {} --name {} --namespace {} {} {}'
+                .format(chart_ref, args.helm_name, args.namespace,
+                        creds_values, helm_args))
+
+def write_temp_credentials(creds_tempfile, creds):
+    creds_tempfile.write("imageCredentials" +
+                    "    username: " + creds[0] +
+                    "    password: " + creds[1])
+    creds_tempfile.flush()
+
+def import_credentials(args):
+    creds = (os.environ.get('LIGHTBEND_COMMERCIAL_USERNAME'),
+             os.environ.get('LIGHTBEND_COMMERCIAL_PASSWORD'))
+
+    if creds[0] == None or creds[1] == None:
+        with open(os.path.expanduser(args.creds), 'r') as creds_file:
+            creds_dict = dict(re.findall(r'(\S+)\s*=\s*(".*?"|\S+)', creds_file.read()))
+            creds = (creds_dict.get('user'), creds_dict.get('password'))
+
+    if creds[0] == None or creds[1] == None:
+        sys.exit("Credentials missing, please check your credentials file\n"
+                 "LIGHTBEND_COMMERCIAL_CREDENTIALS=" + args.creds)
+
+    return creds
+
+def setup_args():
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument('--dry-run', help='only print out the commands that will be executed',
+                        action='store_true')
+    parser.add_argument('--force-install', help='set to true to delete an existing install first, instead of upgrading',
+                        action='store_true')
+    parser.add_argument('--export-yaml', help='export resource yaml to stdout, forces dry_run if not false',
+                        choices=['creds', 'console', 'false'], default='false')
```

Comment:
Maybe "none" instead of "false" since alternative is not "true"?

Reply:
Fuhget my suggestion here.  I see I made that silly choice originally and it's probably not a good idea to change things now that the install script is in the wild.